src/components/Calculator/Calculator.js

```diff
@@ -1,63 +1,64 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { CSSTransitionGroup } from 'react-transition-group'
 
 import BestSubject from './BestSubject'
 import MaxScore from './MaxScore'
+import ScoreTableContainer from '../../containers/ScoreTableContainer'
 import { PRIMARY_SUBJECTS, LINGUAL_SUBJECTS } from '../../constants/SubjectTypes'
 
-const Calculator = ({ maxPrimaryScoreWrapper, maxLingualScoreWrapper }) => {
-  const maxPrimaryScore = maxPrimaryScoreWrapper.computedScore
-  const maxLingualScore = maxLingualScoreWrapper.computedScore
+const Calculator = ({ subjects, maxScoreWrapper, isScoreTableOpened, onToggleClick }) => {
+  const maxPrimaryScore = maxScoreWrapper.primary.max.score
+  const maxLingualScore = maxScoreWrapper.lingual.max.score
 
-  if (maxPrimaryScoreWrapper.isEmpty && maxLingualScoreWrapper.isEmpty) return null
+  if (!subjects.primary.length && !subjects.lingual.length) return null
   return (
     <div className='m-4'>
       <h3>Wynik</h3>
       <hr />
       {!!maxPrimaryScore &&
       <BestSubject
-        maxScoreWrapper={maxPrimaryScoreWrapper}
+        maxScoreWrapper={maxScoreWrapper}
         subjectType={PRIMARY_SUBJECTS}
       />}
       {!!maxLingualScore &&
       <BestSubject
-        maxScoreWrapper={maxLingualScoreWrapper}
+        maxScoreWrapper={maxScoreWrapper}
         subjectType={LINGUAL_SUBJECTS}
       />}
       <br />
       <MaxScore
         maxPrimaryScore={maxPrimaryScore}
         maxLingualScore={maxLingualScore}
       />
+      <br />
+      <button
+        className='btn btn-secondary'
+        onClick={onToggleClick}
+      >
+        {isScoreTableOpened ? 'Ukryj tabelę wyników' : 'Pokaż tabelę wyników'}
+      </button>
+      <CSSTransitionGroup transitionName='scoreTable' transitionEnterTimeout={300} transitionLeaveTimeout={300}>
+        {isScoreTableOpened
+          ? <ScoreTableContainer key='1' />
```

Comment:
Is using '&&' (`{ifSomething && <div>Sth</div>}`) prefered to ternary like:
`{ifSomething ? <div>Sth</div> : null}`?